docs/gettinginvolved/acceptance-testing.rst

```diff
@@ -226,15 +226,15 @@ The nodes should be configured to allow key based SSH connections as user ``root
 
 .. _client-acceptance-tests:
```

Comment:
Moved the client tests off the acceptance testing page, and into the separate client testing page.